src/main/java/be/lghs/accounting/configuration/SecurityUserService.java

```diff
@@ -11,28 +13,49 @@
 
 import java.util.Map;
 import java.util.UUID;
+import java.util.stream.Collectors;
 
 @Service
 @RequiredArgsConstructor
 public class SecurityUserService extends DefaultOAuth2UserService {
 
     private final UserRepository userRepository;
 
+    private static final Logger logger = LoggerFactory.getLogger(SecurityUserService.class);
```

Comment:
We use lombok to create loggers so there is no way to miss copy/pasta errors.

Example:
https://github.com/LgHS/accounting/blob/cab29082c77d7a1871839baf2cb6ddfe54066c6b/src/main/java/be/lghs/accounting/services/UserAccountNumberService.java#L17
https://github.com/LgHS/accounting/blob/cab29082c77d7a1871839baf2cb6ddfe54066c6b/src/main/java/be/lghs/accounting/services/UserAccountNumberService.java#L37

Reply:
done